website/docs/docs/deploy/deploy-environments.md

```diff
@@ -186,6 +190,37 @@ This section allows you to determine the credentials that should be used when co
 </WHCode>
 
 
+## Staging environment <Lifecycle status='beta' />
+
+:::note
+Currently in limited availability beta.
```

Comment:
Do we want to put contact info so customers can ask to join, or is this limited to select clients for the time being?